conf/supervisord.conf

```
@@ -5,7 +5,7 @@ logfile=/dev/null
 logfile_maxbytes=0
 
 [program:web]
-command=newrelic-admin run-program pserve conf/production.ini
+command=newrelic-admin run-program gunicorn --paste conf/production.ini --bind 0.0.0.0:8001
```

```suggestion
command=newrelic-admin run-program gunicorn --paste conf/production.ini
```

Why do we need to specify this again, if it's in the `gunicorn.conf.py`?

---

I don't really want to go through the bother of re-testing this as it requires testing the Docker image locally which involves a bunch of faff because `make run-docker` is really broken, but IIRC the `8001` from `gunicorn.conf.py` wasn't getting picked up in Docker. Maybe `gunicorn.conf.py` doesn't get copied into the Docker image? Or maybe it's not in the CWD when the app is run in the Docker image?